src/nwb2bids/_command_line_interface/_main.py

```diff
@@ -121,3 +122,11 @@ def _run_convert_nwb_dataset(
 
     if console_notification != "" and not silent:
         rich_click.echo(message=console_notification)
+
+    not_any_failures = not notifications or not any(
+        notification.severity == Severity.ERROR for notification in notifications
```

Comment:
What do we do with the case `notification.severity == Severity.CRITICAL`

Reply:
Those more directly relate to the case where `nwb2bids` produces an 'invalid' BIDS directory

I think in a follow-up we can maybe warn that this might be the case, though until various BEPs get merged on BIDS side, this warning will nearly always occur so maybe I'd not add that until things stabilize a bit upstream

Reply:
Sounds good - it is a separate enhancement to the main topic of this PR however (and would benefit from even more targeted improvements + testing on separate PR) opened https://github.com/con/nwb2bids/pull/193